store-armhf.json

```diff
@@ -16,5 +16,13 @@
     "fprocess": "figlet",
     "network": "func_functions",
     "repo_url": "https://github.com/openfaas/faas/tree/master/sample-functions/figlet"
+  },
+  {
+    "title": "Manifest Query",
+    "description": "Docker (@estesp) Manifest Query Tool on OpenFaaS",
```

Comment:
How about a condensed version of Phil's description on his repo?

> A simple utility and backend for querying Docker v2 API-supporting registry images and reporting on "manifest list" multi-platform image support.

adding .. "by Phil Estes"?

Reply:
Thanks.  I've done that.  I was caught between wanting/needing to credit and wanting to keep it brief.